app/assets/stylesheets/splash.css

```diff
@@ -1,14 +1,41 @@
 .splash {
+  height: 100vh;
+  background: lightgray;
+  display: flex;
+  justify-content: space-between;
   flex-direction: column;
+  text-align: center;
+  text-decoration: dashed;
+}
+
+.splash-title {
+  margin-top: 40vh;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+}
```

Comment:
Good progress! However, there's a specific detail to address. Currently, for the `splash-title`, the color is set to black. According to the design, it's not pure black. I recommend adjusting the color to align more closely with the design specifications. Try to match the color as accurately as possible to enhance the visual consistency of your application. Keep up the good work! 🚀🎨